ui/components/telemetry/grafana/GrafanaDateRangePicker.js

```diff
@@ -14,61 +14,75 @@ import {
   FormControlLabel,
   Switch,
   IconButton,
-} from '@material-ui/core';
+  styled,
+} from '@layer5/sistent';
 import Moment from 'react-moment';
 import AccessTimeIcon from '@material-ui/icons/AccessTime';
-import { withStyles } from '@material-ui/core/styles';
 import PropTypes from 'prop-types';
 import MesheryDateTimePicker from '../../MesheryDateTimePicker';
 import { Close } from '@material-ui/icons';
+import { UsesSistent } from '@/components/SistentWrapper';
 
-const styles = (theme) => ({
-  rangeDialog: {
-    // width: window.innerWidth * 0.7,
+const StyledDialog = styled(Dialog)(({ theme }) => ({
+  '& .MuiDialogContent-root': {
+    padding: theme.spacing(2),
   },
-  rangeDialogRow: {
-    display: 'flex',
-    gap: '4rem',
-    '&>*': { flex: '1' },
-  },
-  dateTimePicker: {
-    marginTop: theme.spacing(2),
-    marginBottom: theme.spacing(1),
-  },
-  innerGrid: {
-    borderTop: `1px solid ${theme.palette.divider}`,
-    paddingTop: theme.spacing(2),
-    marginTop: theme.spacing(2),
-  },
-  timeList: {
-    display: 'flex',
-    flexDirection: 'column',
-    alignItems: 'flex-start',
-    '& *': { textAlign: 'left' },
-  },
-  dialogTitleBar: {
-    display: 'flex',
-    justifyContent: 'space-between',
-    backgroundColor: 'rgb(57, 102, 121)',
-    color: 'white',
-  },
-  close: {
-    transform: 'rotate(-90deg)',
-    '&:hover': {
-      transform: 'rotate(90deg)',
-      transition: 'all .3s ease-in',
-    },
-  },
-  closeButton: {
-    height: '2.3rem',
-    marginTop: '0.8rem',
-    marginRight: '10px',
-    color: 'white',
+}));
+
+const DialogTitleBar = styled('div')(() => ({
+  display: 'flex',
+  justifyContent: 'space-between',
+  backgroundColor: 'rgb(57, 102, 121)',
+  color: 'white',
+}));
+
+const CloseIconButton = styled(IconButton)(() => ({
+  height: '2.3rem',
+  marginTop: '0.8rem',
+  marginRight: '10px',
+  color: 'white',
+}));
+
+const CloseIcon = styled(Close)({
+  transform: 'rotate(-90deg)',
+  '&:hover': {
+    transform: 'rotate(90deg)',
+    transition: 'all .3s ease-in',
   },
-  space: { margin: theme.spacing(1) },
-  rangeButton: { border: '1px solid rgba(0, 0, 0, 0.23)' },
 });
 
+const RangeButton = styled(Button)(() => ({
+  border: '1px solid rgba(0, 0, 0, 0.23)',
+}));
+
+const RangeDialogRow = styled('div')(() => ({
+  display: 'flex',
+  gap: '4rem',
+  '& > *': { flex: '1' },
+}));
+
+const DateTimePicker = styled(MesheryDateTimePicker)(({ theme }) => ({
+  marginTop: theme.spacing(2),
+  marginBottom: theme.spacing(1),
+}));
+
+const InnerGrid = styled(Grid)(({ theme }) => ({
+  borderTop: `1px solid ${theme.palette.divider}`,
```

Comment:
how this theme is working if using sistent because this particular color is in meshery. Use sistent border color
```suggestion
  borderTop: `1px solid ${theme.palette.border.<>}`,
```